### PR TITLE
LIVE-1665: add viewports (with addon-essentials) to storybook 

### PR DESCRIPTION
**Pull request by joecowton1**
## Why are you doing this?

Adds viewports to Storybook for Small Mobile, Mobile, Tablet and Tablet Landscape.

This allows us to view any breakpoint at the click of a button - its already highlighted a few small layout bugs that will need addressing.

## Changes

- Update storybook config and add addon

## Screenshots

| Before | After |
| --- | --- |
|  |  |

| Before | After |
| --- | --- |
|  |  |
Screenshot 2021-02-12 at 12 00 55

### CODE REVIEW

**Comment by faresite** [comment]
How about we add `@storybook/addon-essentials` instead? This is what I'm trying to use for the gallery image story as I also need the `backgrounds`

File: .storybook/config.js [hunk 1]
```diff
@@ -1,3 +0,0 @@
-import { configure } from '@storybook/react';
-
-configure(require.context('../src', true, /\.stories\.tsx$/), module);
```
Comment (faresite) [inline .storybook/config.js:3]: Is the configure from `@storybook/react` not needed then?

Comment (joecowton1) [reply .storybook/config.js:3]: No not anymore 👍

File: .storybook/preview.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        width: '1024px',
+        height: '963px',
+        },
+    },
+};
+
+export const parameters = {
+    layout: 'fullscreen',
+    viewport: {
+        viewports: {
```
Comment (JamieB-gu) [inline .storybook/preview.js:18]: You may want to use our list of standard breakpoints [from Source](https://github.com/guardian/source/blob/main/src/core/foundations/src/breakpoints.ts).

**Review by JamieB-gu — COMMENTED** [review]
Fancy!

FYI @SiAdcock @oliverlloyd @sndrs

File: .storybook/preview.js (hunk 2 above)
Comment (joecowton1) [reply .storybook/preview.js:53]: Good call  - added :)

**Comment by joecowton1** [comment]
> How about we add `@storybook/addon-essentials` instead? This is what I'm trying to use for the gallery image story as I also need the `backgrounds`

Added now

**Comment by oliverlloyd** [comment]
Oh wow, there's so much great working happening in apps-rendering at the moment!

**Review by faresite — APPROVED** [review]

**Pull request merged**